src/components/Statistics/index.css

```diff
@@ -0,0 +1,77 @@
+.statisticBox {
```

Comment:
Looks good to me. The only thing which is probably should be different is the class names for different kinds of colors and it should keep the same convention.
success, danger, warning etc. Cause it will make this component very generic so you can use it in any place for any sorts of info.
Also, icon should be implemented using after pseudo-element, not an actual div :) 
I will change that myself and you will see later.